engine/apps/api/serializers/channel_filter.py

```diff
@@ -218,3 +220,15 @@ def update(self, instance, validated_data):
             raise BadRequest(detail="Filtering term of default channel filter cannot be changed")
 
         return super().update(instance, validated_data)
+
+
+class ChannelFilterRetrieveResponseSerializer(ChannelFilterUpdateSerializer):
```

Comment:
maybe `ChannelFilterUpdateResponseSerializer` would be a better name here? this way it indicates that it's a response from update/create, not retrieve